components/camel-google-storage/src/generated/resources/google-storage.json

```diff
@@ -0,0 +1,14 @@
+{
+  "other": {
+    "kind": "other",
+    "name": "google-storage",
+    "title": "Google Storage",
+    "description": "Camel Component for Google Cloud Platform Storage",
+    "deprecated": false,
+    "firstVersion": "3.8.0",
```

Comment:
This will be re-generated again once firstVersion will be 3.9.0

Reply:
done